.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/CONTRIBUTING.md and developer guide https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/docs/dev.md
```

Comment:
nit but if there isn't a `2.` do we need the `1.` ?

Reply:
the purpose of the 1. was so if you thought of another, it could be added. If the review finds nothing to add, I shall remove it! :) 👍